packages/framework-provider-aws/src/library/read-model-adapter.ts

```diff
@@ -20,9 +20,10 @@ export async function fetchReadModel(
   readModelName: string,
   readModelID: UUID
 ): Promise<ReadModelInterface> {
-  const params = {
+  const params: DynamoDB.DocumentClient.GetItemInput = {
     TableName: config.resourceNames.forReadModel(readModelName),
     Key: { id: readModelID },
+    ConsistentRead: true,
```

Comment:
I don't think we should use consistent reads for read models. This makes reading the read models slower, and they're basically caches, so it should not matter if you read the previous version of it, everything else should be designed to allow read models to be eventually consistent

```suggestion
```

Reply:
I think you are right. I wrote this because I was having a lot of failures in integration tests. I don't remember if this was one of the reasons or not 😓

Reply:
Maybe we could configure it differently for integration and for production somehow?